plotsearch/views/plot_search.py

```diff
@@ -244,6 +248,12 @@ class InformationCheckViewSet(
     filterset_class = InformationCheckListFilterSet
 
 
+class DirectReservationLinkViewSet(viewsets.ModelViewSet):
+    queryset = DirectReservationLink.objects.all()
+    serializer_class = DirectReservationLinkSerializer
+    permission_classes = (MvjDjangoModelPermissionsOrAnonReadOnly,)
```

Comment:
So this view is publicly readable?

Reply:
No, this is used by virkailija-ui for DirectReservationLink crud-operations. There is another ApiView for public usage.

Reply:
Well, permission_class should probably be then MvjDjangoModelPermissions